bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/api/Car.java

```diff
@@ -151,6 +157,19 @@ public void setHVACStatus(JsonObject responseJson) {
         }
     }
 
+    public void setPauseStatus(JsonObject responseJson) {
+        try {
+            JsonObject attributes = getAttributes(responseJson);
+            if (attributes != null) {
+                if (attributes.get("action") != null) {
+                    pausemode = PauseMode.valueOf(attributes.get("action").getAsString());
```

Comment:
Will this work? `PauseMode` has been redefined to contain members `ON` and `OFF`, where it was previously `PAUSE` and `RESUME`. I assume the API still returns `PAUSE` and `RESUME`?